tests/drivers/build_all/virtio/testcase.yaml

```
@@ -0,0 +1,12 @@
+common:
+  filter: CONFIG_DT_HAS_VIRTIO_PCI_ENABLED
+  build_only: true
+  tags:
+    - drivers
+    - virtio
+tests:
+  drivers.virtio_pci.build: {}
+  drivers.virtio_pci.mmu.build:
+    filter: CONFIG_CPU_HAS_MMU
+    extra_configs:
+      - CONFIG_MMU=y
```

`MMU` is non longer a user-configurable option and can be thought of as semantically equivalent to `CONFIG_CPU_USE_MMU`. E.g. just because a CPU has an MMU does not mean the platform has configured and enabled the MMU for use.

```suggestion
    filter: CONFIG_MMU
```